data/gui/window/language_selection.cfg

```diff
@@ -135,8 +138,8 @@ If you are a first-time player, we recommend using a mostly-complete translation
 						link_aware = true
 						wrap = true
 						# po: Beneath the label widget displaying this text there is a widget displaying
-						# po: a decorated link to <https://gettext.wesnoth.org/>.
-						label = _ "<b>Note:</b> The translation percentages shown apply to the core game interface and in-game help only. More complete stats are available at the following page:"
+						# po: a clickable link to the wiki’s WesnothTranslations page (which opens in the user’s browser).
+						label = _ "If you are interested in translating the game yourself, we suggest visiting the following page for more information. However, we recommend playing most of the mainline campaigns before considering joining the translators."
```

Comment:
Perhaps rather: "If you are interested in translating the game yourself, follow the link below for more information."

Not sure I agree with the recommendation. If it's an established translation with barely anything missing maybe.

Reply:
So the change from original (before the PR) would be to drop the "if you are a first-time player" sentence completely?

I feel it's like the "good first issue" tags, where some people jumped straight in, without playing the game first. I feel there's a value to telling people to slow down on that decision - if they don't enjoy the game enough to finish most (but not all) of the campaigns, I think they won't stay for the long term.

Reply:
IMO "If you are a first-time player, we recommend using a mostly-complete translation." is something the player can best decide for themselves with the available info. For someone that is proficient in english for example they might not care if the translation is not so complete.

Regarding people jumping in and then not staying... You really think that sentence will solve that issue?

For a translator even if they just play one campaign for the first time and simultaneously translate they might do a good job IMO. If they don't it's up to the translation team to try to fix that or outright not take their contribution. And if they don't finish the campaign they wasted their own work. Not much you can do about that. May be a learning experience for them.

Reply:
To be clear while I think the general issue of people just jumping in without much thought/not finishing etc is simply unsolvable. I don't mean to say we cannot try to mitigate it.

However to me a note suggesting it may be required to play most campaigns before you can help with translations is more likely to be taken seriously by people that would do good work (even if they never play most campaigns and don't stick around long term) and make those not help out even though they perfectly well could.

Reply:
> Perhaps rather: "If you are interested in translating the game yourself, follow the link below for more information."

I took that suggestion, adding a "please" to it.